indico/modules/categories/models/categories.py

```diff
@@ -175,6 +175,11 @@ def __table_args__(cls):
         nullable=False,
         default=False
     )
+    default_ticket_template_id = db.Column(
+        db.ForeignKey('indico.designer_templates.id'),
+        nullable=True,
```

Comment:
But why default to False if its an Integer field? And also in the code I check whether the `default_ticket_template` is None and I even set it to None when deleting the template, so should this `default_ticket_template_id` be non-nullable ?